tests/nlu/base/test_evaluation.py

```diff
@@ -52,6 +52,13 @@
 from rasa.nlu.test import is_response_selector_present
 
 
+@pytest.yield_fixture(scope="session")
+def event_loop(request):
+    loop = asyncio.get_event_loop_policy().new_event_loop()
+    yield loop
+    loop.close()
+
+
```

Comment:
why is this needed here again? From what I understood, it's only used if you pass the `event_loop` fixture into a test function, or the `@pytest.mark.asyncio` decorator is used

Reply:
We have to redefine it because `tests/core/conftest.py` is not imported to `tests/nlu/`